src/wutil/tests.rs

```diff
@@ -72,7 +72,7 @@ fn test_wwrite_to_fd() {
         assert!(fd.is_valid());
         let mut input = WString::new();
         for _i in 0..size {
-            input.push(rng.gen());
+            input.push(rng.r#gen());
```

Comment:
I think this means we can delete `.rustfmt.toml` (enabled by the commit that upgrades the edition, so let's put it in the same commit to make that obvious) which allows me to drop https://github.com/fish-shell/fish-shell/pull/11992/commits/0856141e9902922adb3dbd0cf573d621e2a746c6